tasks/create-advisory/create-advisory.yaml

```diff
@@ -124,9 +125,7 @@ spec:
           exit 1
         fi
 
-        # at this point, one of foundPendingRepositories or foundProdRepositories
```

Comment:
Why remove this comment? I think it was useful. Maybe I would just add "exactly" - "exactly one of"

Reply:
Sure, I am adding it back. I was thinking of making it shorter.

Reply:
It was fixed, and the comment can be resolved. Thanks.